### DISCUSSION TITLE
Backup of Dashboards & Lock Configuration

### DISCUSSION THREAD

**Discussion by sthakkar** [community/community#12987] [Projects and Issues]
We heavily are using dashboards for our issue tracking.

Is there a way to create backups of dashboards (and the values for fields like Priority / Component / Status etc.)?

We had a situation where someone in the team must have accidentally deleted the Priority column thinking they were hiding the column from their view. However, it delete the column for everyone and deleted the corresponding data?

Is there a permission we can apply such that only select users can modify a dashboard?

**Comment by sthakkar** [comment]
Is there an approach we can try for this?

**Comment by tay1orjones** [comment]
Duplicate of https://github.com/community/community/discussions/13373
